### PR TITLE
Unhandled exception

### PR DESCRIPTION
**Pull request by pjanck**
Expanded the IfcGeometryConverter with an Exception for unsupported entities. This should now sufficiently cover the refactor.

### CODE REVIEW

**Comment by pjanck** [comment]
Fixes #24 .

File: Core/src/IfcGeometryConverter/UnhandledException.h [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ * OpenSceneGraph Public License for more details.
+*/
+
+// More details can be found in README.md file in the root directory.
+
+#pragma once
+#ifndef UNHANDLED_REPRESENTATION_EXCEPTION_H
+#define UNHANDLED_REPRESENTATION_EXCEPTION_H
```
Comment (hechth) [inline Core/src/IfcGeometryConverter/UnhandledException.h:21]: Please use better include guards in the future.

File: EarlyBinding/src/EXPRESS/EXPRESSEntity.h [hunk 2]
```diff
@@ -46,10 +46,6 @@ class EXPRESSEntity : public EXPRESSObject {
 		return "#" + std::to_string(m_id);
 	}
 
-	const std::string getErrorLog() const {
```
Comment (hechth) [inline EarlyBinding/src/EXPRESS/EXPRESSEntity.h:49]: Why did you remove this?

**Review by hechth — APPROVED** [review]

**Review by hechth — CHANGES_REQUESTED** [review]
Please also incorporate the newest changes on development in the commits and re-pose the pull request

File: EarlyBinding/src/EXPRESS/EXPRESSEntity.h (hunk 2 above)
Comment (pjanck) [reply EarlyBinding/src/EXPRESS/EXPRESSEntity.h:49]: It was moved to EXPRESSObject

**Pull request merged**